subiquity/server/kernel.py

```diff
@@ -27,3 +32,30 @@ def flavor_to_pkgname(flavor: str, *, dry_run: bool) -> str:
     # that's a bit tricky until we get cleverer about
     # the apt config in general.
     return f'linux-{flavor}-{release}'
+
+
+async def list_installed_kernels(
+        rootfs: pathlib.Path,
+        *, grep_status: str = "grep-status") -> List[str]:
```

Comment:
This argument is unused.

Reply:
Oops!

Reply:
I admit I expected you to fix this by removing the argument :-) What is the use case for passing a different value here?